README.md

```diff
@@ -16,41 +19,48 @@ For tutorials, samples, quick starts and other documentation, visit [Azure for J
 ### Prerequisites
 Java 8 or later is required to use the November 2019 client libraries, otherwise Java 7 or later is required.
 
-## Packages available
+## Release branches (Release tagging)
+
+### Master branch
+The master branch represent more recent code with new features and bug fixes. It does **not** represent latest released **GA** SDK. See [below](#Client-GA-November-2019-Releases) for latest **GA** release.<br/>
+For each package we release there will be a unique git tag created that contains the name and the version of the package to mark the commit of the code that produced the package. This tag will be used for servicing via hotfix branches as well as debugging the code for a particular preview or stable release version.
+Format of the release tags are &lt;package-name&gt;_&lt;package-version&gt;. For more information please [see](https://github.com/Azure/azure-sdk/blob/master/docs/policies/repobranching.md#release-tagging)
+
+## Latest Release
 Each service might have a number of libraries available from each of the following categories:
 
-- [Client - November 2019 Releases](#Client-November-2019-Releases)
+- [Client: GA November 2019 Releases](#Client-GA-November-2019-Releases)
 - [Client - Previous Versions](#Client-Previous-Versions)
 - [Management](#Management)
 
-### Client: November 2019 Releases
-New wave of packages that we are announcing as **GA** and several that are currently releasing in **preview**. These libraries follow the [Azure SDK Design Guidelines for Java](https://azure.github.io/azure-sdk/java/guidelines/) and share a number of core features such as HTTP retries, logging, transport protocols, authentication protocols, etc., so that once you learn how to use these features in one client library, you will know how to use them in other client libraries. You can learn about these shared features [here](core). 
+### [Client: GA November 2019 Releases](#Client-GA-November-2019-Releases)
+New wave of packages that were released in November 2019 client library  as General Availability (GA) and several others that were released in **preview**. These libraries follow the [Azure SDK Design Guidelines for Java](https://azure.github.io/azure-sdk/java/guidelines/) and share a number of core features such as HTTP retries, logging, transport protocols, authentication protocols, etc., so that once you learn how to use these features in one client library, you will know how to use them in other client libraries. You can learn about these shared features [here](sdk/core/README.md). 
```

Comment:
```suggestion
New wave of packages that were released in November 2019 client library as General Availability (GA) and several others that were released in **preview**. These libraries follow the [Azure SDK Design Guidelines for Java](https://azure.github.io/azure-sdk/java/guidelines/) and share a number of core features such as HTTP retries, logging, transport protocols, authentication protocols, etc., so that once you learn how to use these features in one client library, you will know how to use them in other client libraries. You can learn about these shared features [here](sdk/core/README.md). 
```